frontend/src/stores/OutputStore.js

```diff
@@ -152,6 +152,37 @@ class OutputStore {
         }
         return data;
     }
+    @computed get drBayesianPlotData() {
+        const output = this.selectedOutput;
+        const data = [getDrDatasetPlotData(this.selectedDataset)];
+        output.bayesian.models.map(model => {
+            let data2 = {
+                x: model.results.plotting.dr_x,
+                y: model.results.plotting.dr_y,
+                name: model.name,
+                line: {
+                    width: 2,
+                    color: "#FF6352",
```

Comment:
can you pick an array of colors, perhaps from a color palette like this https://www.pinterest.com/pin/589901251166313385/  and assign each model a different color? I guess it could be 5 colors, repeating 2x or whatever makes sense